docs/README.md

```diff
@@ -1,6 +1,24 @@
 # auth-service/Docs
 
 Documentation for the auth service
+## Database
+
+The following Figure depicts the Entity Relationship Diagram of the `Consents` table for the authorization service:
+
+<p align="center">
+  <img src="ErDiagram.png">
+</p>
+
+The following table depicts the types of each attribute in our table, algonside their constraints:
+
+|Attribute| Represented Internal Type | Constraints|
+|:---------:|:---------------------------:|:------------:|
+|`ID`|BIGINT|Unique - Primary Key|
```

Comment:
A couple notes to bring this into line with other Mojaloop standards:
- `ID` should be `id`, and of type `VARCHAR(36)` 
  - take a look at [central-ledger](https://github.com/mojaloop/central-ledger/blob/master/migrations/300200_transfer.js#L32) for example (but don't copy the `tableNameId` pattern, `id` is much better)
  - Also, just for reference: https://github.com/mojaloop/mojaloop-specification/blob/master/Supporting%20Files/fspiop-rest-v1.0-OpenAPI-implementation.yaml#L3061

- Tablenames in the database should be snake case: e.g. `initiator_id`, `participant_id`
  - Knex can map from `snake_case` to `camelCase` for us
- `initiator_id` and `participant_id` should probably also be `VARCHAR(32)`, according to the [swagger spec](https://github.com/mojaloop/mojaloop-specification/blob/master/Supporting%20Files/fspiop-rest-v1.0-OpenAPI-implementation.yaml#L3282)

- Can you go into more detail (perhaps in a different table) about the format of `scopes` and `credential`? I thought that `credential` would be expanded into separate, _nullable_ columns based on our conversation the other day, eg:
```
credential_id:
credential_status: 
credential_type:
credential_challenge:
credential_payload:
```
Alternatively, this could also just be a _nullable_ JSON field as well.

- Finally (and I'm really nitpicking here) the table should probably be called `consent`, and not `Consents`

Reply:
Addressed, and modified, the PR.